ipn/prefs.go

```diff
@@ -570,7 +570,7 @@ func (p *Prefs) AdminPageURL() string {
 		// TODO(crawshaw): In future release, make this https://console.tailscale.com
 		url = "https://login.tailscale.com"
 	}
-	return url + "/admin/machines"
+	return url + "/admin"
```

Comment:
Removing this should be fine - there's only one other caller right now, up CLI text:
https://github.com/tailscale/tailscale/blob/main/cmd/tailscale/cli/up.go#L523

"/machines" is the default admin tab anyways, and it's a little unexpected for this function to return any more than the base URL.